src/admin/containers/email.js

```diff
@@ -76,9 +79,13 @@ const email = toggle_show => ([ address, { content, shown }]) =>
     >
       { shown ? 'Hide' : 'Show' } email
     </button>
-    { shown && <div>{ content.map(line => <p key={line}>{line}</p>) }</div> }
+    { shown && <div>{ content.map(displayEmail) }</div> }
   </li>
 
+const displayEmail = (line, i) => i === 0
```

Comment:
Agreed, we definitely made this decision due to not knowing how we would add a subject key to the object.